csm/cli/schema/bundle_generate.json

```diff
@@ -19,14 +19,6 @@
       "type": "str",
       "help": "Current Node Name."
     },
-    {
```

Comment:
why is this being removed?

Reply:
Not Required anymore Since for Bundle Generate Command I will be parsing os flag as component name. Since the command is hidden